content/docs/graphql/queries/query-documents.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Querying a List of Documents
+id: /docs/graphql/queries/query-documents
+next: /docs/graphql/queries/advanced/filter-documents
+---
+
+List queries offer limited functionality for now.
+
+- Because of the nature of list items, we don't currently auto-generate Tina forms for these queries.
```

Comment:
I know this section was brought over from before, but I don't think I'd know what we mean by: "Because of the nature of list items". Maybe we could leave this point out, (and add a blurb on this to our "useTina" docs?)

Reply:
yeah, I actually don't know what that means either :D

Is this where you are thinking the blurb would go: https://tina.io/docs/tinacms-context/#the-usetina-hook ?

Reply:
Yeah I think it probably makes more sense there (maybe as a blue little info note of something like: 
> Note: Only queries for individual documents can be used with `useTina`. 
?